### PR TITLE
Add events to Cloud Run for Anthos description

### PR DESCRIPTION
**Pull request by grantr**
Now that Cloud Run for Anthos supports eventing (in beta), I added it to the description of the CRfA offering.

## Proposed Changes

- add eventing functionality to CRfA description

### CODE REVIEW

File: docs/knative-offerings.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ vendors for what is or is not supported.
 Here is a list of commercial Knative products (alphabetically):
 
 - [Gardener](https://gardener.cloud/documentation/050-tutorials/content/howto/knative-install/): Install Knative in Gardener's vanilla Kubernetes clusters to add an extra layer of serverless runtime.
-- [Google Cloud Run for Anthos](https://cloud.google.com/run/docs/gke/setup): extend Google Kubernetes Engine with a flexible serverless development platform. With Cloud Run for Anthos, you get the operational flexibility of Kubernetes with the developer experience of Serverless, allowing you to deploy and manage Knative based services on your own cluster.
+- [Google Cloud Run for Anthos](https://cloud.google.com/run/docs/gke/setup): Extend Google Kubernetes Engine with a flexible serverless development platform. With Cloud Run for Anthos, you get the operational flexibility of Kubernetes with the developer experience of Serverless, allowing you to deploy and manage Knative-based services on your own cluster, plus trigger them with events from Google, 3rd-party sources, and your own applications.
```
Comment (abrennan89) [inline docs/knative-offerings.md:17]:
```suggestion
- [Google Cloud Run for Anthos](https://cloud.google.com/run/docs/gke/setup): Extend Google Kubernetes Engine with a flexible serverless development platform. With Cloud Run for Anthos, you get the operational flexibility of Kubernetes with the developer experience of serverless, allowing you to deploy and manage Knative-based services on your own cluster, and trigger them with events from Google, 3rd-party sources, and your own applications.
```

**Review by abrennan89 — CHANGES_REQUESTED** [review]
@grantr suggested minor tweaks :slightly_smiling_face:

**Review by abrennan89 — APPROVED** [review]
/lgtm

**Comment by abrennan89** [comment]
@googlebot I consent.

**Comment by abrennan89** [comment]
@googlebot I fixed it.

**Comment by abrennan89** [comment]
@googlebot I consent

**Comment by abrennan89** [comment]
@grantr hadn't heard back on this one so I committed the suggestion myself, but now I can't fix the CLA. If you fix it this should merge.

**Comment by grantr** [comment]
All contributors have previously signed the CLA.

**Pull request merged**